basics.rb

```diff
@@ -8,9 +8,20 @@
 post '/payload' do
   push = JSON.parse(request.body.read)
   puts "I got some JSON: #{push.inspect}"
-  jsontest = "I got some JSON: #{push.inspect}"
-  Collaborator.add repo_name: "githubschool/open-enrollment-classes-introduction-to-github", issue_num: 927
-  Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
+
+  repo = push["repository"]["full_name"].to_s
+  issue = push["issue"]["number"].to_i
+  user = push["issue"]["user"]["login"].to_s
+
+  if repo === "githubschool/open-enrollment-classes-introduction-to-github" && issue == 927
+    puts "adding to githubschool/open-enrollment-classes-introduction-to-github"
+    Collaborator.add repo_name: "githubschool/open-enrollment-classes-introduction-to-github", issue_num: 927
+  elsif repo ===  "githubschool/on-demand-github-pages" && issue == 1
+    puts "adding to githubschool/on-demand-github-pages"
+    Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
+  else
+    Collaborator.addByIssue repo_name: repo, issue_num: issue, user_login: user
+  end
```

Comment:
@stoe I don't think we actually need this conditional chain, do we? Since we'll ship this along with https://github.com/github/training-kit/pull/441 then issues 927 and 1 won't be used any further.

Aside from that, I think this is ready to go 👍 but will wait till we see you on Monday in case it breaks anything!